shared/common-adapters/clickable-box.js.flow

```diff
@@ -2,10 +2,12 @@
 import React, {Component} from 'react'
 
 export type Props = {
+  children?: any,
   style?: Object,
-  onClick: () => void,
-  hoverColor?: ?string,
-  backgroundColor?: ?string,
+  onClick?: ?() => void,
```

Comment:
i made this change and also added a onPress? void; to all the common adapters that have onClicks